### PR TITLE
docs: Adding m2-reflection.md

### LINKED ISSUE #53: Write reflection on this week's work
rubric={reasoning:12,writing:3}

- In this section, your group should clearly document which parts of your proposal/sketch you have implemented in your dashboard so far and explain what is not yet implemented.
- For anything that you have done differently than in your proposal/sketch explain why (e.g. implementation difficulty, explanation for why the new approach is more effective, etc).
- It is important that you include what you are aware of is not working in your dashboard, so that your TAs can distinguish between features in development and bugs.
- You should also explain if you have intentionally deviated from any of the best practices that we learned in 531 regarding effective visualizations.
- Reflect on what you think your dashboard does well currently what its limitations are, and what are good potential future improvements and additions.
- This section should not be more than 500 words and be written to the file reports/m2-reflection.md in your GitHub repo.

### PR DESCRIPTION
**Pull request by AlysenTownsley**
closes #53

### CODE REVIEW

File: reports/m2-reflection.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Current Implementation
+Our dashboard currently implements the following features from our proposal/sketch:
+- Four Summary Information Cards: These provide a quick overview of key statistics including Overall Death, Death Rate, Percentage of young adults deaths, and Fold Change (2001-2015) (Overall/Young Adults).
```
Comment (SoloSynth1) [inline reports/m2-reflection.md:3]: This is not true anymore ad the key statistics is longer hardcoded to compare stats between 2001 and 2015.

Comment (AlysenTownsley) [inline reports/m2-reflection.md:3]:
```suggestion
- Four Summary Information Cards: These provide a quick overview of key statistics including Overall Death, Death Rate, Percentage of young adults deaths, and Fold Change (Overall/Young Adults).
```

Comment (SoloSynth1) [reply reports/m2-reflection.md]: Resolved in latest commit.

**Review by SoloSynth1 — APPROVED** [review]
LGTM

**Comment by SoloSynth1** [comment]
Thanks @AlysenTownsley, merging this into `main`.

**Pull request merged**